common-auth/SECURITY.md

```diff
@@ -0,0 +1,65 @@
+# Running registry and streamline web-services securely
+
+This module is intended to be used by registry and streamline web-services so that they can enable http client authentication via SPNEGO. Currently supported 
+authentication mechanism is Kerberos. The code for this module has been borrowed from hadoop-auth(2.7.3) module in hadoop project and changed a little. The 
+reasons for doing so are to avoid having a dependency on hadoop-auth module which brings in some other modules, avoid conflicts with other versions of 
+hadoop-auth module and having more control over the changes needed in future. Some text for this document has been borrowed from SECURITY.md of Apache Storm  
+
+By default, registry and streamline web-services are running without authentication enabled and anyone can access the web-services from ui/client as far as they
+know the url and can access the web-server from the client machine. To enable authentication with the client, webservice needs to add a servlet filter from 
+this module. The webservice module will need to declare a dependency on this module. One way of adding a servlet filter in code is as follows. 
+
```

Comment:
This paragraph is just saying that if a webservice needs to use authentication they can use this module. It does not need to declare any dependency on registry top level maven module. Just the auth module here. The code snippet illustrates how you can add a add a servlet filter(which is why it says One way of adding.....) and shows how its done in registry web-service project. Code to add servlet filter is not part of this module. And they can add servlet filters in other ways too. Is it still unclear? Do you still want any modification to the paragraph?